src/css/controls/imports/settings-menu.less

```diff
@@ -5,9 +5,9 @@
     position: absolute;
     flex-flow: column nowrap;
     align-items: flex-start;
-    bottom: @controlbar-height;
+    bottom: 0;
```

Comment:
Making this the default breaks the UI at breakpoints 2+.
![screen shot 2017-09-11 at 2 09 07 pm](https://user-images.githubusercontent.com/4603952/30289761-2eea6664-96fb-11e7-8807-e62fcb781343.png)